### PR TITLE
P3.6 items equal

### PR DESCRIPTION
**Pull request by antgonza**
In this PR:
- re-add line so matplotlib works fine
- self.assertItemsEqual -> self.assertCountEqual
- fix all qiita_core
- fix all processing_job

Note that the biggest differences in networkx versions are:
- before when calling edges/nodes/etc it used to return a dict and now returns a tuple
- all the lists are of the type vs. just a list, for example: nodes, returns NodeView(()) vs []

### CODE REVIEW

File: qiita_pet/webserver.py [hunk 1]
```diff
@@ -239,6 +239,7 @@ def __init__(self):
 
         # 404 PAGE MUST BE LAST IN THIS LIST!
         handlers.append((r".*", NoPageHandler))
+        handlers.append((r"/.*", NoPageHandler))
```
Comment (charles-cowart) [inline qiita_pet/webserver.py]: This line is new, not sure what's supposed to be for. Can you add a comment please?

**Comment by charles-cowart** [comment]
52 files is a lot, but most of the changes are straightforward enough (method name change, slight parameter syntax change, etc.). There is one line where I'd like your comment, just to know what it's doing, and to confirm that it should still be there.

File: qiita_core/tests/test_configuration_manager.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def setUp(self):
         environ['QIITA_CONFIG_FP'] = self.conf_fp
 
         self.conf = ConfigParser()
-        with open(self.conf_fp, 'U') as f:
+        with open(self.conf_fp, newline=None) as f:
```
[the remark this answers is not in the document]
Comment (antgonza) [reply qiita_core/tests/test_configuration_manager.py]: you are right but agree about being explicit ...

File: qiita_pet/webserver.py (hunk 1 above)
Comment (antgonza) [reply qiita_pet/webserver.py]: right, this was a debugging line, removing ....

**Comment by antgonza** [comment]
Thanks for review, I have addressed your comments ...

**Pull request merged**